flytekit/remote/init_remote.py

```diff
@@ -0,0 +1,32 @@
+import typing
+
+from flytekit.configuration import Config
+from flytekit.remote.remote import FlyteRemote
+
+REMOTE_ENTRY = None
+
+
+def init_remote(
+    config: Config,
+    default_project: typing.Optional[str] = None,
+    default_domain: typing.Optional[str] = None,
+    data_upload_location: str = "flyte://my-s3-bucket/",
+    **kwargs,
```

Comment:
I think we should also allow folks to set Options (which have service account etc)

Reply:
I think that `Options` is used for each execution, while `init_function()` is aimed at setting up the `FlyteRemote` object. Should we enable customizing it in the .remote() function instead?

Another option is to set a default Options for `.remote()`. However, I’m unsure if we should also modify the constructor of FlyteRemote for this functionality since it might also be useful in the normal usage of FlyteRemote.

Reply:
i think its ok to modify and set default options. otherwise it will be hard to use with service account. We can do this as a separate PR

Reply:
Roger that